### DISCUSSION TITLE
音乐上传失败

### DISCUSSION THREAD

**Discussion by 2379211424** [Yin-Hongwei/vue-spring-music#140] [Q&A]
添加歌曲失败

**Comment by Yin-Hongwei** [comment]
浏览器F12控制台看报什么错
或者看看后端报什么错
问问题发图片

**Reply by 2379211424** [reply]
![屏幕截图 2022-04-06 173929](https://user-images.githubusercontent.com/95905189/161946733-f5b0eca4-618e-4296-ae15-e68c2d1b0416.png)

**Reply by 2379211424** [reply]
前端传过来文件获取失败

**Reply by Yin-Hongwei** [reply]
浏览器F12控制台看报什么错

**Reply by 2379211424** [reply]
谷歌
![image](https://user-images.githubusercontent.com/95905189/161956025-4310bad8-c3e6-4bc9-8cce-4ee867685892.png)

**Comment by 2379211424** [comment]
![image](https://user-images.githubusercontent.com/95905189/161956327-be6cb0e2-e2b1-40e3-844c-942de7700880.png)

**Reply by Yin-Hongwei** [reply]
你把你上传的音乐和这些信息发我邮箱我试试

**Reply by Yin-Hongwei** [reply]
你上传的音乐有问题，都播放不了，大小0kb。。。

**Comment by 2379211424** [comment]
谢谢
